### ISSUE TITLE
Error when updating Statamic

### ISSUE THREAD

**Issue by jmartsch** [riasvdv/statamic-markdown-highlight#1]
I updated to the latest Statamic 3 beta-32 via `composer update`. While installing I got this error message:
```
Class Rias\MarkdownHighlight\MarkdownHighlight located in P:/htdocs/dotnetic-statamic/vendor/rias/statamic-markdown-highlight/src\Fieldtypes\MarkdownHighlight.php does not comply with psr-4 autoloading standard. Skipping.
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Symfony\Component\Debug\Exception\FatalThrowableError  : Class 'Rias\MarkdownHighlight\MarkdownHighlight' not found

  at P:\htdocs\dotnetic-statamic\vendor\statamic\cms\src\Providers\AddonServiceProvider.php:86
    82|
    83|     protected function bootFieldtypes()
    84|     {
    85|         foreach ($this->fieldtypes as $class) {
  > 86|             $class::register();
    87|         }
    88|
    89|         return $this;
    90|     }

  Exception trace:

  1   Statamic\Providers\AddonServiceProvider::bootFieldtypes()
      P:\htdocs\dotnetic-statamic\vendor\statamic\cms\src\Providers\AddonServiceProvider.php:44

  2   Statamic\Providers\AddonServiceProvider::Statamic\Providers\{closure}()
      P:\htdocs\dotnetic-statamic\vendor\statamic\cms\src\Statamic.php:240

  Please use the argument -v to see more details.
```

**Comment by riasvdv** [comment]
Seems related to this in the changelog:

> [new] Tags, Modifiers, Fieldtypes, and Widgets now automatically register aliases in addons.

I'll take a look as soon as I find some time

**Comment by jmartsch** [comment]
Thank you :)

**Comment by jmartsch** [comment]
It seems to be an error with Composer 2 beta, which I was using. It is working when using Composer 1.10.8 but I get this message while installing:
```
Deprecation Notice: Class Rias\MarkdownHighlight\MarkdownHighlight located in P:/htdocs/dotnetic-statamic/vendor/rias/statamic-markdown-highlight/src\Fieldtypes\MarkdownHighlight.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar://C:/ProgramData/Composer
Setup/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by riasvdv** [comment]
Ah yes, there should be a `\Fieldtypes\` in the namespace, I'll get this fixed soon

**Comment by riasvdv** [comment]
Should be fixed in 1.0.1
